sql/glam-fenix-dev/glam_etl/firefox_desktop_glam_beta__live_v1/query.sql

```diff
@@ -0,0 +1,57 @@
+MERGE INTO
```

Comment:
Instead of doing this in a `query.sql` file we'll need a `script.sql` file here. `query.sql` expect some `SELECT` result to be returned, which is not the case with MERGE INTO